tests/test_workspaces.py

```diff
@@ -20,34 +19,31 @@ def login(prep_properties, pages):
 @allure.story("WorkSpaces Creation and Editing Functionality")
 @allure.severity(allure.severity_level.NORMAL)
 class TestWorkspaces(BaseTest):
-    _JSON_FILE_NAME = "tests_data.json"
 
-    @allure.description("Create new Workspace test")
+    @allure.description("Create new Workspace")
     @allure.title("Create new workspace test")
     @pytest.mark.run(order=1)
     def test_create_new_workspace(self, prep_properties):
```

Comment:
I think you can remove prep_properties from all test methods' signatures, and also change a little bit the code in login method so it will receive an instance of the ini reader and not the whole fixture.
i.e.:

def login (config):
      username = config.config_section_dict("Base Url")["username"]

This is to be changed in workspaces and forgot password tests modules